plugin/src/android/settingsDependency.ts

```diff
@@ -0,0 +1,25 @@
+import { ConfigPlugin, withSettingsGradle } from '@expo/config-plugins'
+
+import { PluginConfigType } from '../pluginConfig'
+
+/**
+ * Update `<project>/settings.gradle` by adding react-native-code-push
+ */
+
+function applySettings(gradleSettings: string) {
+  const codePushSettings = `include ':app', ':react-native-code-push'\nproject(':react-native-code-push').projectDir = new File(["node", "--print", "require.resolve('react-native-code-push/package.json')"].execute(null, rootDir).text.trim(), "../android/app")`
+
+  // Make sure the project does not have the settings already
+  if (!gradleSettings.includes(`include ':app', ':react-native-code-push'`)) {
+    return gradleSettings + codePushSettings
+  }
+
+  return gradleSettings
```

Comment:
Include the `react-native-code-push` settings at the bottom of the file
```suggestion
  const codePushSettings = `\ninclude ':react-native-code-push'\nproject(':react-native-code-push').projectDir = new File(["node", "--print", "require.resolve('react-native-code-push/package.json')"].execute(null, rootDir).text.trim(), "../android/app")`

  // Make sure the project does not have the settings already
  if (!gradleSettings.includes(`include ':react-native-code-push'`)) {
    return gradleSettings + codePushSettings
  }

  return gradleSettings
```